java/CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ add_test(
 
 add_test(
   NAME java-bench-spark-test
-  COMMAND java ${ADD_OPENS} ${JAVA_NIO} -jar bench/spark/orc-benchmarks-spark-${ORC_VERSION}.jar spark data -d sales -c zstd -f orc
+  COMMAND java ${ADD_OPENS} ${JAVA_NIO} -jar bench/spark/orc-benchmarks-spark-${ORC_VERSION}-shaded.jar spark data -d sales -c zstd -f orc
```

Comment:
Sorry, but this is wrong, @cxzl25 . Apache ORC community should use non-shaded version.

Reply:
You described `-nohive `in the PR description like the following, but we don't use `-nohive` at line 100.

May I ask why you did propose to change a working code by giving an irrelevant example? 
```
Error: Invalid or corrupt jarfile spark/target/orc-benchmarks-spark-2.1.0-SNAPSHOT-nohive.jar
```

Reply:
I get an error when I do the following according to the README of bench.

Since `bench/core`, `bench/hive` are packaged with `maven-assembly-plugin`, the jar contains uber, and bench/spark is packaged with `maven-shade-plugin`, the generated jar file name doesn't contain `shaded` in case `shadedArtifactAttached= false`, the generated jar filename does not contain `shaded`.

```bash
cd java
./mvnw clean package -Pbenchmark -DskipTests
cd bench
java -jar spark/target/orc-benchmarks-spark-*.jar spark data
```

```
Error: Invalid or corrupt jarfile spark/target/orc-benchmarks-spark-2.1.0-SNAPSHOT-nohive.jar
```

bench/spark/target
```
-rw-r--r--@ 1 csy  staff   2.1M  1 17 12:03 orc-benchmarks-spark-2.1.0-SNAPSHOT-nohive.jar
-rw-r--r--  1 csy  staff   1.8M  1 17 12:03 orc-benchmarks-spark-2.1.0-SNAPSHOT-shaded-protobuf.jar
-rw-r--r--  1 csy  staff    39K  1 17 12:03 orc-benchmarks-spark-2.1.0-SNAPSHOT-sources.jar
-rw-r--r--  1 csy  staff    13K  1 17 12:03 orc-benchmarks-spark-2.1.0-SNAPSHOT-test-sources.jar
-rw-r--r--  1 csy  staff    13K  1 17 12:04 orc-benchmarks-spark-2.1.0-SNAPSHOT-tests.jar
-rw-r--r--@ 1 csy  staff   217M  1 17 12:04 orc-benchmarks-spark-2.1.0-SNAPSHOT.jar
-rw-r--r--  1 csy  staff    64K  1 17 12:03 original-orc-benchmarks-spark-2.1.0-SNAPSHOT.jar
```

Reply:
@cxzl25 , it sounds like you misunderstood the error message. Technically, your error message is irrelevant with this line because this code is the following which uses exactly `${ORC_VERSION}.jar`.
```
bench/spark/orc-benchmarks-spark-${ORC_VERSION}.jar
```

Please revert this line and leave the documentation revision only.

Reply:
Done.

My original intention is to execute it in * way, so that I don’t need to care about the variable `ORC_VERSION`, just like other benchmarks (` java -jar hive/target/orc-benchmarks-hive-*-uber.jar`) .